cmd/checkpoint/main.go

```diff
@@ -31,25 +31,15 @@ const (
 	kubeAPIServer = "kube-apiserver"
 )
 
+var podAPIServerMeta = unversioned.TypeMeta{
```

Comment:
I want to make it clear that the pod meta is from API sever (which we simulate here) not kubelet. But if you think this is confusing, I can remove the API server part.